library/oneview_fcoe_network.py

```diff
@@ -64,7 +67,10 @@
 
 - name: Update the FCOE network scopes
```

Comment:
Shouldn't the support to OneView be mentioned here also?

Reply:
ok. I will update this file with the comment.